python/ray/tune/trainable.py

```diff
@@ -594,6 +441,8 @@ def _create_logger(self, config, logger_creator=None):
             self._result_logger = logger_creator(config)
             self._logdir = self._result_logger.logdir
         else:
+            from ray.tune.logger import UnifiedLogger
+
```

Comment:
local import to avoid all kinds of circular import problems. Also, it seems like trainable-specific loggers are not really used? I couldn't find a case in the ray code where it has not been instantiated with a noop logger. Should we still keep this for backwards compatibility with custom trainable loggers? Or might this be used by RLLib trainables? cc @ericl